.github/workflows/ci.yml

```diff
@@ -0,0 +1,161 @@
+on:
+  push:
+    tags:
+      # Full version
+      - "[0-9]+.[0-9]+.[0-9]+"
```

Comment:
nit: prefixing with `v` might be a good idea to create a common prefix - it also seems to be a common pattern

Reply:
Generally speaking I'm not a fan of a v prefix. Having the tag match the Cargo version (and also be semver) tends to be quite useful.

Reply:
I've added in the v variants so we have a choice.